ui/src/tool-panel/km-post/dialog/km-post-edit-store.ts

```diff
@@ -287,13 +335,14 @@ export function saveGkPointToEditingGkPoint(point: GeometryPoint): {
 }
 
 export function kmPostGkPointDiffersFromOriginal(
-    editing: Pick<KmPostEditFields, 'gkLocationX' | 'gkLocationY'>,
+    editing: Pick<KmPostEditFields, 'gkLocationX' | 'gkLocationY' | 'gkSrid'>,
```

Comment:
Tuntuisi vähän kyllä jo kaipaavan alityyppiä, tyyliin "GkFields" tms. Tätä voisi hyödyntää muuallakin (sillä näitä kenttien nimiä on jo useammassa paikassa)

Toki tuonkin voisi siis tehdä kahdella tavalla:
```
type GkFields = Pick<KmPostEditFields, 'gkLocationX' | 'gkLocationY' | 'gkSrid'>
```
jolloin funkkarille päätyisi suoraan `editing: GkFields`.

Toisena vaihtoehtona Pick-kutsulle voi antaa nimetyn alityypin:
```
type GkFields = 'gkLocationX' | 'gkLocationY' | 'gkSrid' 
```
jolloin funkkarilla olisi edelleen Pick-kutsu
```
editing: Pick<KmPostEditFields, GkFields>
```
ja tuota `GkFields`-tyyppiä voisi käyttää muissakin tyypeissä.

Oon ainakin ite tottunut käyttämään näitä erikoistyyppejä (kuten Pick) muualla kuin suoraan funkkariparametreissa, mutta ei sinänsä preferenssiä